civictechprojects/static/css/partials/_SubHeader.scss

```diff
@@ -201,7 +201,7 @@
 // @include media-breakpoint-up(sm) {
 // }
 
-@include media-breakpoint-up(md) {
+@media (min-width: 910px) {
```

Comment:
I think we can reduce this amount.  Right before 910 pixels, I can still see a good bit of free space.

![image](https://user-images.githubusercontent.com/322346/55900032-1dc3ff80-5b7b-11e9-9fae-3aaf1061e1f0.png)